.travis.yml

```diff
@@ -5,6 +5,32 @@ language: ruby
 cache: bundler
 
 rvm:
-  - ruby-head
-  - 2.2.2
-  - 2.3.1
\ No newline at end of file
+  - 2.4.1
+  - 2.3.4
```

Comment:
I would remove those and just leave `matrix` in, otherwise it produces too many builds.